python/ribasim/ribasim/input_base.py

```diff
@@ -392,15 +393,24 @@ def check_parent(self) -> "ChildModel":
 class NodeModel(ChildModel):
     """Base class to handle combining the tables for a single node type."""
 
-    _sort_keys: dict[str, list[str]] = {}
-
     @model_serializer(mode="wrap")
     def set_modeld(
         self, serializer: Callable[[type["NodeModel"]], dict[str, Any]]
     ) -> dict[str, Any]:
         content = serializer(self)
         return dict(filter(lambda x: x[1], content.items()))
 
+    @field_validator("*")
+    @classmethod
+    def set_sort_keys(cls, v: Any, info: ValidationInfo) -> Any:
```

Comment:
Could you add some more information in the pull request on how and why you solved the issue for future reference?